### PR TITLE
hw spring: property source, scope, post processor

### PR DESCRIPTION
**Pull request by u2107680007911543774e7r**

### CODE REVIEW

**Review by un-install — COMMENTED** [review]
Ok

**Review by un-install — APPROVED** [review]
Ok

File: src/main/java/homework_spring/DeadMoto.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.stereotype.Component;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+
+@Component
+@PropertySource({"classpath:homework_spring/deadmoto.properties"})
+@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE, proxyMode = ScopedProxyMode.TARGET_CLASS)
+public class DeadMoto {
```
Comment (SerhiiPasko) [inline src/main/java/homework_spring/DeadMoto.java:13]: =)

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**